test/browser/render.js

```diff
@@ -911,5 +911,43 @@ describe('Rendering', function () {
         }
       });
     });
+
+    it('should ignore newer values of old nested observables', function (done) {
```

Comment:
I think a test name like: "should only be concerned with values emitted from a last child observable"